docs/source/advanced/apollo-ast.mdx

```diff
@@ -136,50 +127,54 @@ val schemaIssues = schemaGQLDocument.validateAsSchema()
 println(schemaIssues.map { it.severity.name + ": " + it.message })
 ```
 
-This will print `[WARNING: Unknown directive 'FooBar']`.
+When excuted, this snippet prints `[WARNING: Unknown directive 'private']`.
 
-The `validateAsExecutable` method ensures the operations are consistent with a schema, that is why it takes an instance
-of `Schema` as a parameter. This `Schema` can be obtained by calling the `GQLDocument.toSchema()` extension:
```

Comment:
I'm not sure in Kotlin-land when to refer to something as a method vs. an extension. Is this library adding custom methods to a class that's defined elsewhere?

Reply:
👋  Yes you got it, extensions are when a method (... which are also called functions in Kotlin) is defined not in the class itself, but outside of it. They are used to augment an API you're not the author of (external lib), or sometimes just to organize your code in different modules / files, which is the case here.

To be accurate I used the term extension here, but I think we could have just said method and that would have worked too 😅

Reply:
I guess the proper term is "extension function": https://kotlinlang.org/docs/extensions.html#extension-functions